apps/backend/src/router/index.ts

```diff
@@ -0,0 +1,13 @@
+import { router, publicProcedure } from '../trpc';
+
+export const appRouter = router({
+  health: router({
+    healthcheck: publicProcedure.query(() => {
```

Comment:
Probably have a procedures directory rather than a router directory, since there's only going to be one router

Reply:
Curious how that would look and break out, was planning to follow the merged router method they have in their [docs](https://trpc.io/docs/server/merging-routers), but am open to other ways!